libs/internal/include/launchdarkly/context_filter.hpp

```diff
@@ -48,6 +59,15 @@ class ContextFilter {
         JsonValue& parent;
     };
 
+    /**
+     * Internal delegate function for filter and filter_with_anonymous_redaction.
+     *
+     * @param context The context to redact.
+     * @param redactAnonymous Whether to redact all attributes from anonymous contexts.
+     * @return JSON suitable for an analytics event.
+     */
+    JsonValue filter_(Context const& context, bool redactAnonymous);
```

Comment:
Trailing `_` the preference? Or something like `internalFilter`?

Reply:
Should be able to call it just `filter`, as an overload